test-engineering/load-tests/tests/locustfile.py

```diff
@@ -24,6 +30,61 @@
 # Optional. A comma-separated list of providers to use for this request.
 PROVIDERS: str = ""
 
+# See RemoteSettingsGlobalSettings in
+# https://github.com/mozilla-services/merino/blob/main/merino-settings/src/lib.rs
+KINTO__SERVER_URL = os.environ["KINTO__SERVER_URL"]
+
+# See default values in RemoteSettingsConfig in
+# https://github.com/mozilla-services/merino/blob/main/merino-settings/src/providers.rs
+KINTO__BUCKET = os.environ["KINTO__BUCKET"]
+KINTO__COLLECTION = os.environ["KINTO__COLLECTION"]
+
+# The number of random suggestions stored on each worker
+RS_SUGGESTIONS_COUNT: int = 100
+
+# This will be populated on each worker and
+RS_SUGGESTIONS: List[Dict] = []
+
+
+@events.test_start.add_listener
+def on_locust_test_start(environment, **kwargs):
+    """Download suggestions from Kinto and store random suggestions on workers."""
```

Comment:
Why do workers receive a random set, instead of all of them suggestions?

Reply:
I did this mainly for debugging purposes during development. I'll update this to store all suggestions.